tensorflow_encrypted/protocol/pond.py

```diff
@@ -460,6 +460,22 @@ def mul(self, x, y):
     def square(self, x):
         return self.dispatch('square', x)
 
+    @memoize
+    def mod(self, x, y):
```

Comment:
what do you need mod for?

Reply:
I think I see why, but don't think you need it!

Reply:
relic :)